src/core/electrostatics_magnetostatics/p3m-dipolar.hpp

```diff
@@ -163,7 +161,7 @@ void dp3m_deactivate();
  *  These parameters are stored in the @ref dp3m object.
  *
  *  The function utilizes the analytic expression of the error estimate
- *  for the dipolar P3M method in the paper of J. J. Cerda et al., JCP 2008 in
+ *  for the dipolar P3M method in the paper of @cite cerda08d in
  *  order to obtain the rms error in the force for a system of N randomly
  *  distributed particles in a cubic box.
  *  For the real space error the estimate of Kolafa/Perram is used.
```

Comment:
Just wondering, is there a citation for that one?

Reply:
I love references with the author name but no year and no journal...